Neo4j.Driver/Neo4j.Driver/Internal/Connector/SocketConnection.cs

```diff
@@ -225,6 +216,12 @@ protected virtual void Dispose(bool isDisposing)
             }
         }
 
+        public Task CloseAsync()
+        {
+            Close();
+            return Task.CompletedTask; // TODO verify this is the correct way to do it
```

Comment:
@ali-ince What is the best way to impl a async method that actually do not need to wait for anything?
As you could see, this CloseAsync() on SocketConnection is the same as Close()